src/applications/check-in/components/LanguagePicker.jsx

```diff
@@ -36,17 +24,38 @@ function LanguagePicker() {
           label: 'Español',
           lang: 'es',
         },
-      ].map((link, i) => (
-        <button
-          key={i}
-          onClick={changeLanguage}
-          className={buttonClass(link.lang)}
-          data-testid={`translate-button-${link.lang}`}
-          type="button"
-          value={link.lang}
-        >
-          {link.label}
-        </button>
+        {
+          label: 'Tagalog',
```

Comment:
I think we should probably hide the Tagalog link until we have the translation available? Did we get any guidance on this?

Reply:
makes sense, I just commented it out for now.